### ISSUE TITLE
Crash on launch on Windows 7/10 x64

### ISSUE THREAD

**Issue by jordanmack** [Optiminer/OptiminerZcash#19]
Optiminer v1.3.0 crashes immediately on launch on Windows 7 x64 and Windows 10 x64. Both systems are fully patched with all Windows updates and recent AMD drivers. 

```
'Optiminer.exe' (Win32): Loaded 'C:\Users\username\Desktop\optiminer-zcash-win-1.3.0\Optiminer.exe'. Module was built without symbols.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\ntdll.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\kernel32.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\KernelBase.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\crypt32.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\ucrtbase.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\OpenCL.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\msasn1.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\advapi32.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\user32.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\msvcrt.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\win32u.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\sechost.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\gdi32.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\rpcrt4.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\gdi32full.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\ws2_32.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\psapi.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\mswsock.dll'. Cannot find or open the PDB file.
'Optiminer.exe' (Win32): Loaded 'C:\Windows\System32\imm32.dll'. Cannot find or open the PDB file.
The thread 0x210c has exited with code 0 (0x0).
Unhandled exception at 0x00007FF723EF2734 in Optiminer.exe: Fatal program exit requested.
```

**Comment by Optiminer** [comment]
Please try v1.3.1

**Comment by jordanmack** [comment]
Fixed with v1.3.1.
